src/limb_repo/utils/pybullet_utils.py

```diff
@@ -0,0 +1,13 @@
+"""Utilities for working with PyBullet."""
+
+import pybullet_utils.bullet_client as bc
+
+
+def get_good_joints(p: bc.BulletClient, body_id: int):
+    """Get the joints that are not locked."""
+    good_joints = []
+    for i in range(p.getNumJoints(body_id)):
+        joint_info = p.getJointInfo(body_id, i)
+        if joint_info[2] != 4:  # if not locked
```

Comment:
can we put this in a config somewhere and write a more general function?

Reply:
How would you suggest doing this?

Reply:
oh sorry I misread this before. I thought you were looking up a specific joint index. ignore!